runelite-client/src/main/java/net/runelite/client/util/SwingUtil.java

```diff
@@ -293,4 +299,54 @@ public static void addModalTooltip(AbstractButton button, String on, String off)
 	{
 		button.addItemListener(l -> button.setToolTipText(button.isSelected() ? on : off));
 	}
+
+	/**
+	 * Removes all of a component's children faster than calling removeAll() on it in many cases
+	 */
+	public static void fastRemoveAll(Container c)
+	{
+		// If we are not on the EDT this will deadlock, in addition to being totally unsafe
+		assert SwingUtilities.isEventDispatchThread();
+
+		// when a component is removed is has to be resize for some reason, but only if its valid
```

Comment:
This sentence is kind of spaghetti. Is it meant to say something along the lines of `valid components must be resized when they are removed for some reason`?

Reply:
it is supposed to say
`when a component is removed it has to be resized for some reason, but only if it's valid`